raster-foundry/raster-foundry
# Fix download for MODIS scenes

## Overview

This PR update the MODIS scene factory to unquote the s3 location for the uploaded
geotiff when creating the image and adds a migration to correct the image source URIs
for existing MODIS scenes.

### Checklist


## Testing Instructions

 * If you don't have any projects with MODIS scenes yet:
<details>
<ul>
 <li> make a project </li>
 <li> add a MODIS scene to it </li>
 <li> process the upload </li>
 <li> try to download the MODIS scene you added and confirm that you can't </li>
</ul>
</details>

 * Create a new project with a MODIS scene
 * Process the upload
 * Verify that you can download the MODIS scene from that project
 * Run migrations
 * Try to download the MODIS scene from your old project again

Closes #3665
